test/recovery_to_vault.js

```diff
@@ -64,6 +102,25 @@ contract('Proxy funds', accounts => {
     })
   }
 
+  // Token test groups
+  const tokenTestGroups = [
+    {
+      title: 'standards compliant, reverting token',
+      tokenContract: TokenMock,
+      revertsOnFailure: true,
```

Comment:
Where are these being used?

Reply:
Ah good catch, I must've copied those over from the `safe_erc20` tests and not used them (since everything should revert in `VaultDepositable`)!